src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.storage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
```

Comment:
The `fruitStorage` map should be private to adhere to the encapsulation principle. Public static fields can be modified from anywhere in the code, which could lead to unexpected behavior and makes the code less secure.